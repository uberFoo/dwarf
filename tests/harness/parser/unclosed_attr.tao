#[foo = "bar"
struct Foo {}

fn main() {}