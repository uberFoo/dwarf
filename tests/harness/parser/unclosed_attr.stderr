[31mError:[0m Unclosed delimiter [33m[[0m
   [38;5;246mâ•­[0m[38;5;246mâ”€[0m[38;5;246m[[0munclosed_attr:2:1[38;5;246m][0m
   [38;5;246mâ”‚[0m
 [38;5;246m1 â”‚[0m [33mâ•­[0m[33mâ”€[0m[33mâ–¶[0m[33m [0m[38;5;249m#[0m[33m[[0m[33mf[0m[33mo[0m[33mo[0m[33m [0m[33m=[0m[33m [0m[33m"[0m[33mb[0m[33ma[0m[33mr[0m[33m"[0m
 [38;5;246m2 â”‚[0m [33mâ”œ[0m[33mâ”€[0m[33mâ–¶[0m[33m [0m[31ms[0m[31mt[0m[31mr[0m[31mu[0m[31mc[0m[31mt[0m[38;5;249m [0m[38;5;249mF[0m[38;5;249mo[0m[38;5;249mo[0m[38;5;249m [0m[38;5;249m{[0m[38;5;249m}[0m
 [38;5;240m  â”‚[0m [33mâ”‚[0m   [31mâ”€[0m[31mâ”€[0m[31mâ”€[0m[31mâ”¬[0m[31mâ”€[0m[31mâ”€[0m          
 [38;5;240m  â”‚[0m [33mâ”‚[0m      [31mâ•°[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m Must be closed before this [31mstruct[0m
 [38;5;240m  â”‚[0m [33mâ”‚[0m                   
 [38;5;240m  â”‚[0m [33mâ•°[0m[33mâ”€[0m[33mâ”€[0m[33mâ”€[0m[33mâ”€[0m[33mâ”€[0m[33mâ”€[0m[33mâ”€[0m[33mâ”€[0m[33mâ”€[0m[33mâ”€[0m[33mâ”€[0m[33mâ”€[0m[33mâ”€[0m[33mâ”€[0m[33mâ”€[0m[33mâ”€[0m[33mâ”€[0m[33mâ”€[0m[33mâ”€[0m Unclosed delimiter [33m[[0m
[38;5;246mâ”€â”€â”€â•¯[0m

